code/modules/mob/living/carbon/human/human_attackhand.dm

```diff
@@ -3,22 +3,28 @@
 	if(M == src)
 		return //Can't bite yourself
 
-/*Vampire code
+//Vampire code
 	if(M.zone_sel && M.zone_sel.selecting == LIMB_HEAD && src != M)
-		if(M.mind && isvampire(M) && !M.mind.vampire.draining)
-			if(!M.can_suck(src))
+		if(M.mind && isvampire(M))
+			var/datum/role/vampire/V = M.mind.GetRole(VAMPIRE)
```

Comment:
Note to self : it is more efficient (and works the same) to do
```dm
var/datum/role/vampire/V = isvampire(M)
if (V)
```
It also checks for the mind of M in the meantime